Easydict/App/Localizable.xcstrings

```diff
@@ -6002,6 +6070,74 @@
         }
       }
     },
+    "service.configuration.openai.system_prompt.placeholder" : {
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "You are a helpful translation assistant."
+          }
+        },
+        "en-CA" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "You are a helpful translation assistant."
+          }
+        },
+        "sk" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Jste užitečná pomocí překladu."
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "你是一个有用的翻译助手。"
+          }
+        },
+        "zh-Hant" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "你是一個有用的翻譯助手。"
+          }
+        }
+      }
+    },
+    "service.configuration.openai.system_prompt.title" : {
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "System Prompt"
+          }
+        },
+        "en-CA" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "System Prompt"
+          }
+        },
+        "sk" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "System Prompt"
```

Comment:
`System Prompt` needs translation.

Reply:
I know, but since specialized words like `Prompt` are not easy to translate, I chose to keep both `System Prompt` and `User Prompt`.

Or do you think `系统 Prompt` and `用户 Prompt` are better?

Reply:
`系统 Prompt` and `用户 Prompt` are better.

Reply:
ok, translated.